src/shopDetail/_components/redeemFamilyMembers/redeemFamilyButton/RedeemFamilyButton.tsx

```diff
@@ -34,7 +34,7 @@ export const RedeemFamilyButton = ({ member, onPress, reward }: TProps) => {
           <Styled.UnredeemableModalTitle fontStyle="bold" size="large">
             {t('SHOP_DETAIL.WHO_CAN_REDEEM.UNREDEEMABLE_MODAL_TITLE')}
           </Styled.UnredeemableModalTitle>
-          <Typography bottomSpacing="12px">{error.endUserMessage.nl}</Typography>
+          <Typography bottomSpacing="12px">{!data.redeemable ? data.message : error.endUserMessage.nl}</Typography>
```

Comment:
when do you get an error and when the data message?

Reply:
The endpoint returns a 200 response with `redeemable: false` when you do not have enough points to redeem the reward. In all other cases, it responds with an error instead.